documents/service.go

```diff
@@ -60,6 +60,12 @@ type Service interface {
 
 	// Update validates and updates the model and return the updated model
```

Comment:
This `Create` and `Update` will be removed as soon as we finalized the transition to coreapi svc, right?

Reply:
yeap